src/components/sections/GetStarted.tsx

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import "./GetStarted.css";
 
 const GetStarted: React.FC = () => {
-  const map: any = require("../../../assets/united_states.png");
+  const map: any = require("../../assets/united_states.png");
```

Comment:
turn this require into an import at the top of the file.

Reply:
yeeeee legacy code from someone else

Reply:
we're not even using thisf iled, I'll just kill it